phpBB/includes/mcp/mcp_queue.php

```diff
@@ -1433,6 +1434,8 @@ static public function disapprove_posts($post_id_list, $id, $mode)
 				}
 			}
 
+			$disapprove_reason_lang = $disapprove_reason_lang ?? '';
```

Comment:
```suggestion
			$disapprove_reason_lang = $disapprove_reason_lang ? '';
```
I think there is a Typo.

Best regards 👍

Reply:
No, it's actually as intended, see the explanation of the null coalescing operator: https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op